demo/vl-wizard.html

```diff
@@ -0,0 +1,327 @@
+                                <vl-checkbox id="vl-wizard-checkbox-pane-2-previous" label="Vorige toegelaten"></vl-checkbox>
+                                <vl-checkbox id="vl-wizard-checkbox-pane-2-next" label="Volgende toegelaten"></vl-checkbox>
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane id="vl-wizard-checkbox-pane-3" data-vl-previous-pane-disabled data-vl-next-pane-disabled>
+                        <h3 is="vl-h3" slot="title">Stap 3</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <vl-checkbox id="vl-wizard-checkbox-pane-3-previous" label="Vorige toegelaten"></vl-checkbox>
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Versturen
+                        </button>
+                    </vl-wizard-pane>
+                </div>
+            </vl-wizard>
+            <script>
+                customElements.whenDefined('vl-wizard-pane').then(() => {
+                    const pane1 = document.getElementById('vl-wizard-checkbox-pane-1');
+                    const pane2 = document.getElementById('vl-wizard-checkbox-pane-2');
+                    const pane3 = document.getElementById('vl-wizard-checkbox-pane-3');
+                    document.getElementById('vl-wizard-checkbox-pane-1-next').oninput = event => event.detail ? pane1.enableNextPane() : pane1.disableNextPane();
+                    document.getElementById('vl-wizard-checkbox-pane-2-previous').oninput = event => event.detail ? pane2.enablePreviousPane() : pane2.disablePreviousPane();
+                    document.getElementById('vl-wizard-checkbox-pane-2-next').oninput = event => event.detail ? pane2.enableNextPane() : pane2.disableNextPane();
+                    document.getElementById('vl-wizard-checkbox-pane-3-previous').oninput = event => event.detail ? pane3.enablePreviousPane() : pane3.disablePreviousPane();
+                });
+            </script>
+        </div>
+    </div>
+</body>
+
+</html>
```

Comment:
je sprak dacht ik van een afnemer die validatie "onNext" wou doen, zouden we best dan niet dat in de demo-pagina steken?

Reply:
Dat was voor wanneer we de vereenvoudiging van #2 zouden voorzien dat dergelijke API nodig zou zijn, nu kan de gebruiker rechtstreeks met de acties werken.

Reply:
ja, maar zou je daar dan geen voorbeeld van laten zien, als dat het eerst is wat de eerst afnemer er mee wil gaan doen?

Reply:
Ik hoop dat voor het toevoegen van event listeners of andere interactie met een link/button geen demo nodig is in kader van een wizard.

Reply:
wat de afnemer wil gaan bereiken staat los van de implementatie toch? allez, als jij met de huidige demo dat genoeg aangetoond acht is het goed voor mij hoor, ik zou ze gewoon de pap in de mond geven ...

Reply:
Offline besproken, extra demo logica toevoegen is irrelevant in kader van de wizard.